File: AR7ExportToolbar/ToolStripExport/Exporters/TiffSectionReportExporter.cs
using System;
using System.Collections.Generic;
using System.Collections.Specialized;
using System.Linq;
using System.Text;
using GrapeCity.ActiveReports.Document;
using GrapeCity.ActiveReports.Export;

namespace AR7ExportToolbar.ToolStripExport.Exporters
{
	public class TiffSectionReportExporter : SectionReportExporter
	{
		private const string TypeName = "GrapeCity.ActiveReports.Export.Image.Tiff.Section.TiffExport";
		private const string AssemblyName = "GrapeCity.ActiveReports.Export.Image.v7";

		public TiffSectionReportExporter() : base(TypeName, AssemblyName)
		{
		}

		public override void Export(SectionDocument document, string filename, NameValueCollection settings)
		{
			throw new NotImplementedException();
		}

		public override string DefaultExtension
		{
			get { return "tiff"; }
		}

		public override string FileDialogFilter
		{
			get { return "TIFF Files (*.tiff)|*.tiff|All Files (*.*)|*.*"; }
		}
	}
}

File: AR7ExportToolbarVB/ToolStripExport/Exporters/TiffSectionReportExporter.vb
Imports GrapeCity.ActiveReports.Document
Imports System.Collections.Specialized

Public Class TiffSectionReportExporter
	Inherits SectionReportExporter
	Private Const TypeName As String = "GrapeCity.ActiveReports.Export.Image.Tiff.Section.TiffExport"
	Private Const AssemblyName As String = "GrapeCity.ActiveReports.Export.Image.v7"

	Public Sub New()
		MyBase.New(TypeName, AssemblyName)
	End Sub

	Public Overrides Sub Export(document As SectionDocument, filename As String, settings As NameValueCollection)
		Throw New NotImplementedException()
	End Sub

	Public Overrides ReadOnly Property DefaultExtension() As String
		Get
			Return "tiff"
		End Get
	End Property

	Public Overrides ReadOnly Property FileDialogFilter() As String
		Get
			Return "TIFF Files (*.tiff)|*.tiff|All Files (*.*)|*.*"
		End Get
	End Property
End Class